### PR TITLE
Add tenant and namespace getters to Python context

### PR DESCRIPTION
**Pull request by lucperkins**
This PR supplies some missing getters for the Python context object (to achieve parity with the Java Pulsar Functions SDK).

### CODE REVIEW

File: pulsar-client-cpp/python/functions/context.py [hunk 1]
```diff
@@ -50,9 +50,18 @@ def get_message_id(self):
     pass
 
   @abstractmethod
-  def get_topic_name(self):
+  def get_input_topic_name(self):
```
Comment (srkukarni) [inline pulsar-client-cpp/python/functions/context.py]: Can we see if java also has the same interface?

Comment (lucperkins) [reply pulsar-client-cpp/python/functions/context.py]: I updated the corresponding Java method as well. I think that `getTopicName`/`get_topic_name` is slightly confusing given that there are so many topics in play. Happy to revert if you disagree.

Comment (srkukarni) [reply pulsar-client-cpp/python/functions/context.py]: Actually the method applies to the message that is currently being processed. So maybe we should update that to get_current_msg_topic_name or something equivalent?

Comment (lucperkins) [reply pulsar-client-cpp/python/functions/context.py]: Good idea. I'll update.

**Pull request merged**